<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_R">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_G">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_B">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
